cabal.project.bootstrap

```diff
@@ -0,0 +1,12 @@
+packages:
+    Cabal
+  , Cabal-syntax
+  , Cabal-hooks
+  , cabal-install
+  , cabal-install-solver
+
+-- Don't include tests or benchmarks for bootstrapping
+tests: False
+benchmarks: False
+
+index-state: hackage.haskell.org 2024-04-22T06:16:57Z

```

Comment:
Sigh. I will report about this duplicated index state and the breakage that necessitated this new file in https://github.com/haskell/cabal/issues/9901